add markAsCopy and dynamicPTable to config.md (#233)
contao/docs: docs/dev/reference/dca/config.md

| label              | `&$GLOBALS['TL_LANG']` (`string`) | The label is used with page or file trees and typically includes reference to the language array.                                               |
| ptable             | Parent table (`string`)           | Name of the related parent table (table.pid = ptable.id).                                                                                       |
| dynamicPtable      | true/false (`bool`)               | Dynamically set the parent table like in `tl_content`.                                                                                          |
| ctable             | Child tables (`array`)            | Name of the related child tables (table.id = ctable.pid).                                                                                       |
| dataContainer      | Data Container (`string`)         | Table (database table), File (local configuration file) or Folder (file manager).                                                               |

> | markAsCopy         | Field name (`string`)             | Mark the new record with "copy of" in this field name. Since Contao 4.6                                                                         |

Reviewer: ```suggestion
| markAsCopy         | Field name (`string`)             | Appends "(copy)" to this field when copying a record (Contao __4.6__ and up).                                                                         |
```

Reviewer: Ich kann leider nicht direkt in deinen branch comitten, müsstest du selbst comitten ;)
